helm/ph-ee-engine/ams-paygops/templates/deployment.yaml

```diff
@@ -19,6 +19,11 @@ spec:
 {{ toYaml .Values.deployment.annotations | indent 8 }}
 {{- end }}
     spec:
+      initContainers:
+      #During this Pod's initialization, check that zeebe-gateway service is up and running before starting this pod
+      - name: check-zeebe-gateway-ready
+        image: busybox:latest
+        command: ['sh', '-c','until nc -vz g2p-sandbox-zeebe-gateway 26500; do echo "Waiting for myconfig service"; sleep 2; done;']
```

Comment:
`g2p-sandbox-` is coming from the release name. if you hard code this word, it is going to be difficult to manage the chart. use {{Release.name}} if possible

Reply:
Done